static/templates/filings/candidate.hbs

```diff
@@ -4,83 +4,66 @@
     <span class="panel__subtitle t-bold"><a href="{{basePath}}/committee/{{committee_id}}">{{ committee_name }}</a></span>
   </div>
   <table>
-    <tr>
-      <td class="panel__term" scope="row">Report coverage</td>
-      <td class="panel__data">{{datetime coverage_start_date pretty=true}} - {{datetime coverage_end_date pretty=true}}</td>
-    </tr>
-    <tr>
-      <td class="panel__term" scope="row">Receipt date</td>
-      <td class="panel__data">{{ datetime receipt_date pretty=true }}</td>
-    </tr>
-    <tr>
-      <td class="panel__term" scope="row">Amendment indicator</td>
-      <td class="panel__data">{{ decodeAmendment amendment_indicator }}</td>
-    </tr>
+    {{#panelRow "Report coverage"}}
+      {{datetime coverage_start_date pretty=true}} - {{datetime coverage_end_date pretty=true}}
```

Comment:
You didn't introduce this, but I just caught that this needs to be `format=pretty`.